models/choral.js

```diff
@@ -65,6 +65,20 @@ choralSchema.statics.createNew = function (attrs, cb) {
   });
 };
 
+choralSchema.statics.findAllForUser = function (user, cb) {
+  console.log("user id: " + user._id);
+  this.find({ userId: user._id }, (err, chorals) => {
+    if (err) {
```

Comment:
I really like the 

if (err) return cb(err, null);
cb(null, err);

rather than the if else. Just my opinion